tests/interfaces/records-read.spec.ts

```diff
@@ -22,6 +22,51 @@ describe('RecordsRead', () => {
 
       expect(recordsRead.message.descriptor.messageTimestamp).to.equal(currentTime);
     });
+
+    it('should reject if `recordId`, `protocol` and `protocolPath` are all missing', async () => {
+      const alice = await TestDataGenerator.generatePersona();
+      const readPromise = RecordsRead.create({
+        authorizationSignatureInput: Jws.createSignatureInput(alice),
+      });
+
+      await expect(readPromise).to.be.rejectedWith(DwnErrorCode.RecordsReadMissingCreateProperties);
+    });
+
+    it('should not reject if only `recordId` is passed', async () => {
+      const alice = await TestDataGenerator.generatePersona();
+      const readPromiseSuccess = RecordsRead.create({
+        recordId                    : 'some-id',
+        authorizationSignatureInput : Jws.createSignatureInput(alice),
+      });
+
+      await expect(readPromiseSuccess).to.not.be.rejected;
```

Comment:
Nit: Rather than just testing that it doesn't reject, can you check that `recordId` is present once it resolves?